StyleCop.Analyzers/StyleCop.Analyzers.Test/Helpers/UseCultureAttribute.cs

```diff
@@ -97,6 +100,9 @@ public override void Before(MethodInfo methodUnderTest)
         /// <param name="methodUnderTest">The method under test</param>
         public override void After(MethodInfo methodUnderTest)
         {
+            CultureInfo.DefaultThreadCurrentCulture = this.originalCulture;
+            CultureInfo.DefaultThreadCurrentUICulture = this.originalUiCulture;
+
```

Comment:
The `CultureInfo.DefaultThreadCurrentCulture` and `CultureInfo.DefaultThreadCurrentUICulture` should be reset to their original values (which is most likely `null` and not the currently active culture.)